### DISCUSSION TITLE
Future downloadable small vision sound  models

### DISCUSSION THREAD

**Discussion by hpssjellis** [huggingface/huggingface.js#100] [Ideas]
Not sure if it fits the HuggingFace business model but there could be some use for a few small vision sound models to be downloaded to the desktop computer. The strong argument is data security but what I am interested in is speed.

**Comment by coyotte508** [comment]
you can do this with the python lib, find a small model among the 100k+ models on https://huggingface.co/ - see https://huggingface.co/tasks

It's python however, but see https://github.com/huggingface/huggingface.js/issues/82 too

**Reply by hpssjellis** [reply]
> you can do this with the python lib, find a small model among the 100k+ models on https://huggingface.co/ - see https://huggingface.co/tasks
> 
> It's python however, but see #82 too

I am looking for Javascript clientside. I got squeeznet running about 4 years ago   [here](https://rocksetta.com/tensorflowjs/pre-trained/31-squeezenet.html), but no worries, HuggingFace is amazing.
